packages/bot-web-ui/src/stores/transactions-store.js

```diff
@@ -113,4 +130,9 @@ export default class TransactionsStore {
     clear() {
         this.elements = this.elements.slice(0, 0); // force array update
     }
+
+    @action.bound
+    disposeTransactionListeners() {
+        this.disposeTransactionsListener();
+    }
```

Comment:
We could call this directly rather than wrapping it in a function with the same name.

Reply:
Names are not the same, the function's is plural. I wanted to dispose it in `run-pannel-store.js` so I created an action for that.

Reply:
I see what you mean now. 

Another approach would be to define an `onUnmount` in the `TransactionsStore` which will be called from `run-panel-store.js`, then we could avoid this confusing name altogether.